distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -531,8 +569,31 @@ private static Integer findPortNumber(Integer portToTry) {
 
     /**
      * Allows extending classes to add any custom options to the configuration.
+     * This is only valid for tests run with the PerClass strategy
      */
     protected Option[] configureCustom() {
+        try {
+            return options(// extra config options for catalog-ui
+                    installStartupFile(getClass().getResourceAsStream("/catalog-ui/users.properties"),
+                            "/etc/users.properties"),
+                    installStartupFile(getClass().getResourceAsStream("/catalog-ui/users.attributes"),
+                            "/etc/users.attributes"),
+                    wrappedBundle(maven().groupId("io.fastjson")
```

Comment:
This should be in the itest-dependencies-app features.xml. Boon is also a bundle, no need to wrap it.